verification/src/http_server/routers/solidity.rs

```diff
@@ -31,8 +31,8 @@ impl Router for SolidityRouter {
             .app_data(self.cache.clone())
             .service(
                 web::scope("/verify")
-                    .route("/multi-part", web::post().to(multi_part::verify))
-                    .route("/standard_json", web::post().to(standard_json::verify)),
+                    .route("/multiple", web::post().to(multi_part::verify))
+                    .route("/json", web::post().to(standard_json::verify)),
```

Comment:
let's keep it `standard_json` may be, because it kinda a default name for the specific json for the compiler input

i.e. if a person sees `json`, he thinks "what json is it?", but with `standard_json` he probably will instantly recognize compiler input

also `standard_json` is [easily googlable](https://www.google.com/search?q=solidity+standard+json&oq=solidity+standard+json&aqs=chrome.0.69i59j0i512l2.3833j0j7&sourceid=chrome&ie=UTF-8)